src/pages/Beta/Airdrop2/index.tsx

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components'
+import { ChainId } from '@pangolindex/sdk'
+
+const ModalUpper = styled(DataCard)`
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.1);
+  background: radial-gradient(76.02% 75.41% at 1.84% 0%, #ffc800 0%, #e1aa00 100%);
+  padding: 0.5rem;
+`
+const AirdropUI: React.FC = () => {
+  const { account, chainId } = useActiveWeb3React()
+  const [eligible, setEligible] = useState<boolean>(false)
+
+  const [changeMyChain, setChangeChain] = useState<boolean>(false)
+
+  const [modalOpen, setModalOpen] = useState<boolean>(false)
+
+  const canClaim = useUserHasAvailableClaim(account)
+  const claimAmount = useUserUnclaimedAmount(account)
+  const amount = claimAmount?.toFixed(0, { groupSeparator: ',' })
+  const { claimCallback } = useClaimCallback(account)
+
+  const checkStatus = () => {
+    if (Number(amount) > 0) {
+      if (canClaim) setEligible(true)
+      else setModalOpen(true)
+    } else {
+      setModalOpen(true)
+    }
+  }
+
+  const changeChain = () => {
+    setChangeChain(true)
+  }
+
+  const claimPNG = () => {
+    if (Number(amount) > 0) claimCallback()
+  }
+  const renderBoxesWagmi = () => {
+    if (!account && !eligible && !changeMyChain) {
+      return <BoxNotConnected />
+    }
+    if (account && !eligible && !changeMyChain) {
+      return <BoxChangeChain changeChain={changeChain} />
+    }
+    if (account && changeMyChain && !eligible) {
+      if (chainId === ChainId.WAGMI) return <BoxCheckEligibility checkStatus={checkStatus} />
+      else return <BoxChangeChain changeChain={changeChain} />
+    }
+    if (account && changeMyChain && eligible) {
+      return <BoxClaimReward claimPNG={claimPNG} amount={amount} />
+    } else {
+      return <></>
+    }
+  }
+
+  function wrappedOnDismiss() {
+    setModalOpen(false)
+  }
+  const renderError = (isOpened: boolean) => {
+    return (
+      <Modal isOpen={isOpened} onDismiss={wrappedOnDismiss} maxHeight={250} minHeight={30} isBeta={true}>
+        <ModalUpper>
+          <CardBGImage />
+          <CardNoise />
+          <ColumnCenter>
+            <Text
+              fontSize={25}
+              fontWeight={500}
+              lineHeight="50px"
+              color="black"
+              style={{ textAlign: 'center', paddingTop: '30px' }}
+            >
+              Sorry, you are not eligible
+            </Text>
+            <PngTokenAnimated width="55px" src={tokenLogo} />
+          </ColumnCenter>
+        </ModalUpper>
+      </Modal>
+    )
+  }
+
+  return (
+    <PageWrapper>
+      <Box paddingBottom="20px">
+        <span style={{ textAlign: 'center' }}>
+          <MainTitle>Pangolin Going Crosschain</MainTitle>
+          <Text fontSize={18} fontWeight={500} lineHeight="27px" color="text10">
+            And we are not empty handed!
+          </Text>
+        </span>
+      </Box>
+      <BoxWrapper>
+        {renderBoxesWagmi()}
+        <Confetti start={Boolean(eligible)} />
+        <ClaimBox>
+          <span
```

Comment:
move all inline styles from this file to separate styled component, please refer how we have implemented other pages. apply this type of change to every file in this PR